plugins/woocommerce-admin/client/marketing/components/CardHeaderTitle/CardHeaderTitle.scss

```diff
@@ -0,0 +1,6 @@
+.woocommerce-marketing-card-header-title {
+	font-size: 20px;
+	font-weight: 400;
+	line-height: 28px;
+	letter-spacing: 0;
+}
```

Comment:
💅  What about using some variables here?

Reply:
@puntope , thanks, I simplified the code in [`c2aad26` (#36540)](https://github.com/woocommerce/woocommerce/pull/36540/commits/c2aad264734ebe858a44cef9c6597037b6d77b80):

- Use `@include font-size ( 20 )` to generate relative size in rem.
- Use unitless number value for line-height; this is the preferred way according to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height).
- `font-weight` and `letter-spacing` are not needed.

The UI looks the same before and after this change.